setup.sh

```
@@ -0,0 +1,111 @@
+#!/bin/bash
+# SAOS Sovereign CLI Forge - Zachary Dakota Hulse
+# No money. No cloud. No compromise.
+
+set -e
+
+BRYER_MODE=false
+LOCAL_BUILD=false
+
+for arg in "$@"; do
+  case $arg in
+    --bryer-mode) BRYER_MODE=true ;;
```

Shellcheck (0.11.0)

[warning] BRYER_MODE appears unused. Verify use (or export if used externally).

(SC2034)